test/aks-version.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"strings"
+)
+
+var (
+	upgradeProfile = flag.String("upgrade-profile", "", "UpgradeProfile of aks. Will be always sorted by aks version.")
+)
+
+//AksProfiles for available aks version to install and upgrade
+type AksProfiles []struct {
+	Upgrades []Upgrades `json:"upgrades"`
+	Version  string     `json:"version"`
+}
+
+//Upgrades profiles for given aks version
+type Upgrades struct {
+	IsPreview           bool   `json:"isPreview"`
+	Orchestratortype    string `json:"orchestratorType"`
+	Orchestratorversion string `json:"orchestratorVersion"`
+}
+
+func main() {
+	flag.Parse()
+
+	aksVersions := []string{}
+	currentVersion := ""
+
+	aksProfiles := AksProfiles{}
+	json.Unmarshal([]byte(*upgradeProfile), &aksProfiles)
```

Comment:
We should handle errors and fail if something goes wrong

Reply:
file removed